src/manifests_workflow/input_manifests.py

```diff
@@ -11,6 +11,8 @@
 from abc import abstractmethod
 from typing import Dict, List, Type, Union
 
+import ruamel.yaml
```

Comment:
The code now reads and writes YML files in two different ways. We need to have one.